libs/designsystem/src/lib/components/avatar/avatar.component.spec.ts

```diff
@@ -184,8 +184,12 @@ describe('AvatarComponent', () => {
 
```

Comment:
This change _seems_ unrelated, but the approach chosen here for looping over each color broke when I introduced the overlay variants in  `libs/core/src/scss/themes/_colors.scss`.

Instead it is now aligned with the actual implementation inside the component.